SecretSanta/src/SecretSanta.Api/Controllers/PairingController.cs

```diff
@@ -0,0 +1,38 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Threading.Tasks;
+using AutoMapper;
+using Microsoft.AspNetCore.Mvc;
+using SecretSanta.Api.ViewModels;
+using SecretSanta.Domain.Services.Interfaces;
+
+namespace SecretSanta.Api.Controllers
+{
+    public class PairingController : ControllerBase
+    {
+        private IPairingService PairingService { get; set; }
+        private IMapper Mapper { get; set; }
```

Comment:
These properties could be made readonly.
```suggestion
        private IMapper Mapper { get; }
```